opm/material/densead/Evaluation.hpp

```diff
@@ -47,26 +48,26 @@ namespace DenseAd {
  * \brief Represents a function evaluation and its derivatives w.r.t. a fixed set of
  *        variables.
  */
-template <class ValueT, unsigned numVars>
```

Comment:
My experience is that gcc can optimize nothing better than int. And then unsigned is also unspecific. So what is the problem with int? I don't get any warnings and I always use -Wall as in all warnings.